src/main/java/org/broadinstitute/hellbender/tools/reference/CompareReferences.java

```diff
@@ -141,6 +171,28 @@ public void traverse(){
         for(ReferencePair pair : referencePairs){
             System.out.println(pair);
         }
+
+        if(referencePairs.size() != 1 && baseComparisonMode != BaseComparisonMode.NO_BASE_COMPARISON){
```

Comment:
move these sanity checks to before the tool starts doing any of the work, (perhaps in "custom commandline validation"